libbeat/kibana/client.go

```diff
@@ -283,9 +293,37 @@ func (client *Client) GetDashboard(id string) (common.MapStr, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error removing index pattern: %+v", err)
 	}
+
+	// remove unsupported chars
+	objects := result["objects"].([]interface{})
+	for _, obj := range objects {
+		o := obj.(common.MapStr)
+		for _, key := range responseToDecode {
+			// All fields are optional, so errors are not caught
+			decodeValue(o, key)
```

Comment:
We ignore all values, I wonder if we could or should log them in debug mode?
I am trying to think about the future me and if question on discuss could be solved by that.